### PR TITLE
SUSE readme added

### PR DESCRIPTION
**Pull request by msmeissn**
#### Ticket Link

Fixes

#### Release Note

```release-note

```

### CODE REVIEW

**Comment by lukehinds** [comment]
Thanks! I wonder if it might be worth mentioning kubewarden , cc @flavio ?

**Comment by flavio** [comment]
It's all in the open, it can be added.

@msmeissn: can you take care of that as part of this PR or do you want me to follow up once this gets merged?

File: SUSE/README.md [hunk 1]
```diff
@@ -3,3 +3,7 @@
 - [SUSE](https://www.suse.com) is implementing SigStore methods in its Container and Product deliveries.
```
Comment (flavio) [inline SUSE/README.md]: nitpicking, using the same naming conventions adopted by the sigstore project:

```suggestion
- [SUSE](https://www.suse.com) is implementing sigstore methods in its Container and Product deliveries.
```

File: SUSE/README.md [hunk 2]
```diff
@@ -3,3 +3,7 @@
...
 - The [SUSE Container Registry](https://registry.suse.com/) is now also signing the SUSE containers using "cosign" (additionaly to Notary and Atomic methods) and also uploading signatures to the Linux Foundation Transparency Rekor log.
```
Comment (flavio) [inline SUSE/README.md:4]: nitpicking, using the same naming conventions adopted by the sigstore project. Plus addressing a small typo:

```suggestion
- The [SUSE Container Registry](https://registry.suse.com/) is now also signing the SUSE containers using Cosign (additionally to Notary and Atomic methods) and also uploading signatures to the Linux Foundation Transparency Rekor log.
```

**Review by flavio — CHANGES_REQUESTED** [review]
Thanks @msmeissn for adding kubewarden to the list.

**Comment by flavio** [comment]
@msmeissn can you also comply with the DCO check please? If that's a trouble I can submit the commit with you as a co-author

**Comment by lukehinds** [comment]
> @msmeissn can you also comply with the DCO check please? If that's a trouble I can submit the commit with you as a co-author

I can toggle that on as admin, I just need you to say you agree with the DCO

https://wiki.linuxfoundation.org/dco

**Comment by msmeissn** [comment]
i rebased the commits into 1 with signed-off

I agree to the DCO.

**Pull request merged**